### ISSUE TITLE
Can't install latest release 

### ISSUE THREAD

**Issue by shinobisoft** [contour-terminal/contour#531]
Tried installing on Ubuntu 20.4 with `apt install ...`. Apt informed me that I had an unmet dependency, `contour-terminfo`.

**Comment by christianparpart** [comment]
Hey. Did you install contour or contour-cli? Where did you get the files from?

**Comment by shinobisoft** [comment]
Sorry, I downloaded the latest .deb from your Release section. "contour_0.2.2.175-ubuntu_20_04_amd64.deb"

**Comment by christianparpart** [comment]
@shinobisoft btw, we've just release version 0.2.3, which you can now also get from the GH release page. That includes the .deb dependency fix.

**Comment by shinobisoft** [comment]
Thanks. I downloaded it yesterday. Still the same issue tho, missing contour-terminfo. Tried building locally too. No issues with the build other than a couple of warnings. But when I run my build it crashes. Something to do with OpenGl... That could be an issue on my end as my PC is older than dirt! 😂 I'm looking to replace my PC soon. But I WILL be keeping an eye on this project!!! BTW, thanks for your hard work!!!
